### ISSUE TITLE
When `frame-cmds` is loaded, activating ivy-hydra heads causes the minibuffer to be deselected.

### ISSUE THREAD

**Issue by bcc32** [syl20bnr/spacemacs#14492]
#### Description :octocat:
When `frame-cmds` is loaded, activating ivy-hydra heads causes the
minibuffer to be deselected.

A pull request to fix frame-cmds is on the way.

#### Reproduction guide :beetle:
- Start Emacs
- `SPC z f +` to zoom frame in
- `SPC f f` to pull up some random ivy prompt
- `C-o t` (`hydra-ivy/ivy-toggle-marks`)

*Observed behaviour:* :eyes: :broken_heart:
The selected window changes, and the minibuffer is no longer the
current buffer.  The ivy candidates are not marked with ">" until the
minibuffer is selected manually.

*Expected behaviour:* :heart: :smile:
The ivy candidates are immediately marked with ">", and the minibuffer
remains selected.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 27.1
- Spacemacs: 0.300.0
- Spacemacs branch: nil (rev. b7cbcb5ed)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: hybrid
- Completion: ivy
- Layers:
```elisp
(spell-checking
 (syntax-checking :variables syntax-checking-use-original-bitmaps t)
 (auto-completion :variables auto-completion-idle-delay nil)
 (ivy :variables ivy-enable-advanced-buffer-information t)
 helpful ibuffer
 (org :variables org-projectile-file "TODO.org" org-want-todo-bindings t)
 (unicode-fonts :variables unicode-fonts-enable-ligatures t)
 autohotkey c-c++ csv nil emacs-lisp
 (go :variables go-format-before-save t go-tab-width 4)
 haskell html
 (javascript :variables javascript-fmt-tool 'prettier javascript-fmt-on-save t)
 json bibtex
 (latex :variables latex-build-command "latexmk")
 markdown perl5 python ruby rust nil shell-scripts sql
 (typescript :variables typescript-fmt-tool 'prettier typescript-fmt-on-save t)
 yaml nixos osx git
 (version-control :variables version-control-diff-tool nil)
 spacemacs-completion spacemacs-layouts spacemacs-editing spacemacs-editing-visual spacemacs-evil spacemacs-language spacemacs-misc spacemacs-modeline spacemacs-navigation spacemacs-org spacemacs-project spacemacs-purpose spacemacs-visual command-log fasd nil
 (shell :variables shell-enable-smart-eshell t)
 spotify)
```
- System configuration features: XPM JPEG TIFF PNG CAIRO SOUND DBUS GSETTINGS GLIB NOTIFY INOTIFY GNUTLS LIBXML2 FREETYPE HARFBUZZ ZLIB TOOLKIT_SCROLL_BARS LUCID X11 XDBE XIM MODULES THREADS JSON PDUMPER GMP

**Comment by duianto** [comment]
There's a PR pending to update `frame-cmds.el`:
[bot] Auto-update #14511

**Comment by bcc32** [comment]
Yep, thanks.  Is there a way to link this issue to that PR so that this will automatically be closed?

**Comment by duianto** [comment]
If a linking keyword followed by the issue number is used in a pull request description or a commit message for example:
`Fixes #14492`
or:
`Closes #14492`
then the issue is closed when the PR is merged (when the PR is on the default branch).

source:
https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

Or an issue can be choosen from `Linked issues` in the right panel.
That's what I just did and it added the note above.

Haven't tried it before, let's see what happens when it's merged 😄

**Comment by duianto** [comment]
It worked.
The PR's changes have now been applied to the `develop` branch:
[bot] Auto-update #14511
